src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
-
```

Comment:
All your other classes have 1 empty line after class declaration but here you removed it.
Please decide on code-style.
Either all classes have this 1 empty line or all of them don't have it

Reply:
Fix this please